### DISCUSSION TITLE
Fulltext search: is it possible to search a part of a word in content item title?

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#7230] [Q&A]
Hi all, 

probably I miss something. I have correctly configured the fulltext search but now, to find a contentitem, I have to search it by words . I would like to search content item using a part of the title with not complete words. Is it possible?

Thank you.

**Comment by Skrypt** [comment]
You need to set your TitlePart indexation to "Analyzed" and not "Stored".

**Reply by PiemP** [reply]
Hi @Skrypt,

I have unchecked "Stored" and rebuild the index but seems doesn't work.

Probably I'm not explained well. I would like search a content item using a filter like "wor" and found items that contains "world" or "word" in the items title. 

I have see the `Allow Lucene queries in search forms` options in the Search configuration and, with a term like "wor*", I can do my job: but I would like to have something that doesn't need wildcard.

PS. I'm not sure, but I have looked at the fulltext documentation and I believe the bolded part is wrong:

>Index latest version : this option will allow you to index only published items or also index drafts which could be useful if you want to search for content items in a custom frontend dashboard or even in an admin backend custom module. By default **if we don't check this option it will only index published content items**

**Reply by Skrypt** [reply]
Rebuild you index if you check / uncheck this option. It won't do it for you. And if you previously indexed drafts they will remain in the index unless you rebuild it. Else it should do what is written.

For the other request I believe that if you want to run a custom Lucene Query that uses a Wildcard Query instead of a Term Query you can't. I might take a look at the options we have later on. Right now we wanted to offer a basic search that people can extend at their will. Nothing more.
